docs/source/catalog.rst

```diff
@@ -190,3 +190,8 @@ Whether a particular catalog entry supports direct or proxied access is determin
 - ``force``: Force all clients to access the data directly.  If they do not have the required plugin, an exception will be raised.
 
 Note that when the client is loading a data source via direct access, the catalog server will need to send the plugin arguments to the client.  Do not include sensitive credentials in a data source that allows direct access.
+
+Client Authorization Plugins
+''''''''''''''''''''''''''''
+
```

Comment:
Should include the path to the simple auths we have and/or a very brief description of what they are expected to provide (a headers method).